nixos/modules/services/misc/owncast.nix

```diff
@@ -0,0 +1,100 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let cfg = config.services.owncast;
+in {
+
+  options.services.owncast = {
+
+    enable = mkEnableOption "Whether to enable owncast";
```

Comment:
The "Whether to enable" text is already included by `mkEnableOption` so you can remove please.